### PR TITLE
https://swisschain.atlassian.net/browse/SIR-186

### PR DESCRIPTION
**Pull request by OttoVT**

### CODE REVIEW

File: src/Integrations.Common/Domain/Blockchains/Blockchain.cs [hunk 1]
```diff
@@ -1,14 +1,39 @@
-﻿using Swisschain.Sirius.Sdk.Primitives;
+﻿using System;
+using Swisschain.Sirius.Sdk.Primitives;
 
 namespace Integrations.Common.Domain.Blockchains
 {
+    /*
```
Comment (KonstantinRyazantsev) [inline src/Integrations.Common/Domain/Blockchains/Blockchain.cs]: This comment looks excess here

File: src/Integrations.Common/Domain/Protocols/Protocol.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+﻿using Swisschain.Sirius.Sdk.Primitives;
+
+namespace Integrations.Common.Domain.Protocols
+{
+    public sealed class Protocol
+    {
+        public Protocol(BlockchainId id)
```
Comment (KonstantinRyazantsev) [inline src/Integrations.Common/Domain/Protocols/Protocol.cs]: BlockchainId -> ProtocolId

File: src/Integrations.Common/Persistence/INetworksRepository.cs [hunk 3]
```diff
@@ -1,12 +1,12 @@
 ﻿using System.Collections.Generic;
 using System.Threading.Tasks;
-using Integrations.Common.Domain.Networks;
+using Integrations.Common.Domain.Blockchains;
 
 namespace Integrations.Common.Persistence
 {
-    public interface INetworksRepository
+    public interface IBlockchainsRepository
```
Comment (KonstantinRyazantsev) [inline src/Integrations.Common/Persistence/INetworksRepository.cs]: The filename doesn't match the type name

File: src/Integrations.Common/Persistence/NetworksRepository.cs [hunk 4]
```diff
@@ -2,49 +2,49 @@
...
 using System.Linq;
 using System.Threading.Tasks;
-using Integrations.Common.Domain.Networks;
+using Integrations.Common.Domain.Blockchains;
 using Swisschain.Sirius.Sdk.Primitives;
 
 namespace Integrations.Common.Persistence
 {
-    public sealed class NetworksRepository : INetworksRepository
+    public sealed class BlockchainsRepository : IBlockchainsRepository
```
Comment (KonstantinRyazantsev) [inline src/Integrations.Common/Persistence/NetworksRepository.cs]: The filename doesn't match the type name

File: src/Integrations.Common/Persistence/NetworksRepository.cs [hunk 5]
```diff
@@ -2,49 +2,49 @@
...
+            _store = new Dictionary<BlockchainId, Blockchain>
             {
-                [("Bitcoin", "RegTest")] = new Network(
-                    "RegTest", 
-                    "Bitcoin", 
-                    "Reg test", 
+                ["bitcoin-regtest"] = new Blockchain(
+                    "bitcoin-regtest", 
+                    "bitcoin", 
+                    "Reg Test",
```
Comment (KonstantinRyazantsev) [inline src/Integrations.Common/Persistence/NetworksRepository.cs]: Should be "Bitcoin - Reg Test"

File: src/Integrations.Common/Persistence/NetworksRepository.cs [hunk 6]
```diff
@@ -2,49 +2,49 @@
...
+                    "Reg Test", 
                     NetworkType.Private, 
                     new Uri("http://bitcoin-api-testnet.services.svc.cluster.local")),
-                [("Ethereum", "Ropsten")] = new Network(
-                    "Ropsten", 
-                    "Ethereum", 
+                ["ethereum-ropsten"] = new Blockchain(
+                    "ethereum-ropsten", 
+                    "ethereum", 
                     "Ropsten",
```
Comment (KonstantinRyazantsev) [inline src/Integrations.Common/Persistence/NetworksRepository.cs]: Should be "Ethereum - Ropsten"

File: src/Integrations.Common/Domain/Blockchains/Blockchain.cs [hunk 7]
```diff
@@ -1,14 +1,39 @@
...
+            Name = name;
+            Type = type;
+            IntegrationUrl = integrationUrl;
+            TenantId = tenantId;
         }
 
         public BlockchainId Id { get; }
+        public ProtocolId ProtocolId { get; }
+        public string Name { get; }
+        public NetworkType Type { get; }
```
Comment (KonstantinRyazantsev) [inline src/Integrations.Common/Domain/Blockchains/Blockchain.cs]: Should be `NetworkType`

File: src/Integrations.Worker/Program.cs [hunk 8]
```diff
@@ -35,6 +35,7 @@ public static void Main(string[] args)
             new HostBuilder()
                 .SwisschainService<Startup>(options =>
                 {
+                    options.UsePorts(5100, 5101);
```
Comment (KonstantinRyazantsev) [inline src/Integrations.Worker/Program.cs]: Should be in `#if DEBUG`

File: src/Integrations.Common/Domain/Blockchains/Blockchain.cs (hunk 1 above)
Comment (OttoVT) [reply src/Integrations.Common/Domain/Blockchains/Blockchain.cs]: fixed

File: src/Integrations.Common/Domain/Protocols/Protocol.cs (hunk 2 above)
Comment (OttoVT) [reply src/Integrations.Common/Domain/Protocols/Protocol.cs]: fixed

File: src/Integrations.Worker/MessageConsumers/PublishModelMessageConsumer.cs [hunk 9]
```diff
@@ -46,7 +46,7 @@ public async Task Consume(ConsumeContext<PublishModel> context)
                 {
                     BlockchainId = blockchain.Id,
                     Name = blockchain.Name,
-                    Type = NetworkTypeMapper.ToMessagingContract(blockchain.Type),
+                    Type = NetworkTypeMapper.ToMessagingContract(blockchain.NetworkType),
```
Comment (KonstantinRyazantsev) [inline src/Integrations.Worker/MessageConsumers/PublishModelMessageConsumer.cs]: Property names in the event should match entity property names

**Pull request merged**